### DISCUSSION TITLE
muting & disconnecting other users from voice chat

### DISCUSSION THREAD

**Discussion by Poirot12345T** [Rapptz/discord.py#6843] [Q&A] [answered]
Hi,
due to lack of shortcut/keybind that'll disconnect user from a voice chat I decided that I'll write a bot that'll disconnect people from the voice chat when they ask to (by connceting to voice called "disconnect", but that's another story) - is there any possibility of using moderation tools like disconecting others with a bot in a discord.py library? Thanks for the answer!
Poirot12345T

**Comment by apple502j** [comment, marked as answer]
Maybe `await member.move_to(None)`?

https://discordpy.readthedocs.io/en/latest/api.html#discord.Member.move_to

**Reply by Poirot12345T** [reply]
Thank you, I'll try ❤️ 
Edit 22:24 CEST: It works! Thank you!
